### PR TITLE
docs: add link to git.md

### PR DESCRIPTION
**Pull request by liya-oz**
New link with concise guidelines on writing clear, human- and machine-readable commits[Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/).

### CODE REVIEW

File: git-and-github/git.md [hunk 1]
```diff
@@ -108,6 +108,9 @@ The above simple commands will be about 95% of what you will do with git. As lon
 
 This should give you a better idea of what it is exactly that you want to fix and should help you describe what you want to do. If you can describe your problem well then the internet (or ChatGPT) will always have an answer.
 
+You can also find concise guidelines on writing clear, human- and machine-readable commits on the [Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/).
```
Comment (allhandsondeck) [inline git-and-github/git.md:111]: What is the purpose of dash after the "human" keyword? I might not be seeing it properly as reviewing from my mobile phone.

Comment (yuliiaozkan) [reply git-and-github/git.md:111]: Thank you @allhandsondeck. Yes, you are right.  I meant "human-readable and machine-readable commits", but that might not have been clear enough.  Let me write it in the different way.

"You can also find concise guidelines on writing clear, human-readable and machine-readable commit messages on the [Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/)."

**or** 

"You can also find concise guidelines on writing clear, well-structured commit messages on the [Conventional Commits site](https://www.conventionalcommits.org/en/v1.0.0/)."

**Which one do you find better?**

Comment (stasel) [reply git-and-github/git.md:111]: I like
> "You can also find concise guidelines on writing clear, well-structured commit messages on the Conventional Commits site."

**Review by allhandsondeck — APPROVED** [review]

File: git-and-github/git.md (hunk 1 above)
Comment (yuliiaozkan) [reply git-and-github/git.md]: @stasel I have added an additional commit with updates. Please review the changes.

**Review by stasel — APPROVED** [review]

**Pull request merged**